lock/src/main/java/com/auth0/android/lock/LockActivity.java

```diff
@@ -273,7 +297,59 @@ public void onSocialAuthenticationRequest(SocialConnectionEvent event) {
         CallbackHelper helper = new CallbackHelper(pkgName);
         lastIdp = new WebIdentityProvider(helper, options.getAccount(), idpCallback);
         lastIdp.setUseBrowser(options.useBrowser());
+        lastIdp.setParameters(options.getAuthenticationParameters());
         lastIdp.start(LockActivity.this, event.getConnectionName());
     }
 
+    @SuppressWarnings("unused")
+    @Subscribe
+    public void onDatabaseAuthenticationRequest(DbLoginEvent event) {
+        if (options == null) {
+            return;
+        }
+        String dbConnectionName = null;
+        if (options.getDefaultDatabaseConnection() != null) {
```

Comment:
What if the connection name of options cannot be found in Auth0 configuration, which are the enabled ones?